system-tests/test-binary/ci_environments_spec.ts

```diff
@@ -24,12 +24,6 @@ describe('e2e binary CI environments', () => {
     },
   )
 
-  // TODO: Where is this image located? Needs to be bumped to Node 16.16.0 or later
-  smokeTestDockerImage(
```

Comment:
We aren't sure where this image lives and is just making sure plain xvfb fails in node 12 with cypress. I don't think this test is incredibly valuable to keep around, let alone we don't know what the image is or where its maintained

Reply:
@AtofStryker This image is here. Not sure if you want to update the image or remove. https://github.com/cypress-io/cypress-internal-docker-images